app/views/friend_notifier_mailer/inform.html.erb

```diff
@@ -0,0 +1,3 @@
+Hello <%=@user.first_name%>!
+
+<%= link_to "Visit here to activate your account.", emails_url(@user, host: 'localhost:3000') %>
```

Comment:
This generates an email that you can check on localhost:1080 after typing "mailcatcher" in your termimnal

Reply:
So this allows yo to see the emails even though they arent being officially sent?